src/main/java/org/jvalue/outboxer/AmqpPublisher.java

```diff
@@ -96,6 +97,8 @@ private Message createAmqpMessage(String eventId, String payload) {
     messageProps.setContentType(MessageProperties.CONTENT_TYPE_JSON);
     messageProps.setContentEncoding(StandardCharsets.UTF_8.name());
     messageProps.setMessageId(eventId);
+    // Persist message so it does survive RabbitMQ restarts
+    messageProps.setDeliveryMode(MessageDeliveryMode.PERSISTENT);
```

Comment:
Is there any scenario where we might want to have it non-persistent? Then we should make it configurable via an env variable. If not, having it hard coded should be fine.

Reply:
>Is there any scenario where we might want to have it non-persistent?

I do not think so. Having a non-persistent message means we can not guarantee at least once delivery because RabbitMQ can restart at any time.

If we really want to make it configurable we should not do it via an env var. I would prefer adding new column to the outbox database table, so it can be configured per message.

Reply:
then lets leave it like that for now